Subsystems/cmdUtil/start-test-app.sh

```diff
@@ -0,0 +1,16 @@
+# Command to start the test application
+#  CFE_ES_START_APP_CC = 4
+#
+# typedef struct CFE_ES_StartAppCmd_Payload
+# {
+#      char Application[CFE_MISSION_MAX_API_LEN]; //CFE_MISSION_MAX_API_LEN = 20
+#      char AppEntryPoint[CFE_MISSION_MAX_API_LEN];
+#      char AppFileName[CFE_MISSION_MAX_PATH_LEN]; //CFE_MISSION_MAX_PATH_LEN = 64
+#      CFE_ES_MemOffset_t StackSize; //uint32
+#      CFE_ES_ExceptionAction_Enum_t ExceptionAction; //uint8
+#      undocumented SPARE uint8
+#      CFE_ES_TaskPriority_Atom_t Priority; //uint16
+# } CFE_ES_StartAppCmd_Payload_t;
+
+./cmdUtil --pktid=0x1806 --cmdcode=4 --endian=LE --string="20:CFE_TEST" --string="20:CFE_TestMain" \
```

Comment:
Since you are changing the app name, you will need to update the test that are checking for the app name.

Reply:
Updated the command for CFE_TEST_APP in 3ee0fa